Source/WebKit/UIProcess/API/Cocoa/WKUserContentController.mm

```diff
@@ -146,14 +146,15 @@ - (void)removeAllContentRuleLists
     {
     }
 
-    void didPostMessage(WebKit::WebPageProxy& page, WebKit::FrameInfoData&& frameInfoData, API::ContentWorld& world, WebKit::JavaScriptEvaluationResult&& jsMessage) final
+    void didPostMessage(WebKit::WebPageProxy& page, WebKit::FrameInfoData&& frameInfoData, API::ContentWorld& world, std::optional<WebKit::JavaScriptEvaluationResult>&& jsMessage) final
     {
         @autoreleasepool {
             RetainPtr webView = page.cocoaView();
             if (!webView)
                 return;
+            RetainPtr<id> body = jsMessage ? jsMessage->toID() : NSNull.null;
```

Comment:
nit:

```suggestion
            RetainPtr<id> body = jsMessage.transform([](auto result) {
                return result.toID();
            }).value_or(NSNull.null);
```

Reply:
also, you may be able to drop `<id>` because CTAD

Reply:
I like dropping the template parameters.  I think the optional::transform is a bit verbose for such a simple case where a ternary operator works.